README.md

```diff
@@ -53,3 +53,153 @@ The output values are the rank of the low-rank approximation, assembly time, and
 We only use `sympartialACA` with `PCARegularClustering`, but we use a different number of threads and MPI processes to test scalability.
 
 The output values are the space-saving, assembly time, and the timing of 25 matrix-vector products.
+
+## Task-based implementation benchmark
+
+This section presents the results of our benchmarks on the task-based implementation of our algorithm. We conducted these benchmarks to evaluate the performance of our algorithm and identify areas for improvement. 
+
+The results of these benchmarks are presented in graph format.
+The benchmarks were conducted on different thread configurations and problem sizes, and we measured the execution times and performance of our algorithm. 
+
+In addition, we also provide a detailed explanation of how to use the benchmarks and customize them to suit your specific needs.
+
+### Overview
+Our aim is to test the implementation of task-based parallelism compared with a conventional parallel implementation. 
```

Comment:
we are not only benchmarking the task-based implementation. At the moment, we are benchmarking the hmatrix implementation of htool.